base_classes/NXcg_polyline.nxdl.xml

```diff
@@ -132,52 +110,33 @@ they share the same position in space but have different identifiers.-->
         <doc>
              If true indicates that the vertices are all placed at different
              positions and have different identifiers, i.e. no points overlap
-             or are counted twice.
+             or are counted several times.
         </doc>
     </field>
     <field name="polylines" type="NX_INT" units="NX_UNITLESS">
         <doc>
-             Sequence of vertex identifiers which describe each polyline.
+             Sequence of identifier for vertices how they build each polyline.
              
              A trivial example is a set with two polylines with three vertices each.
              If the polylines meet in a junction, say the second vertex is shared
              and marking the junction between the two polylines, it is possible that
-             there are only five unique positions suggesting five unique vertices.
+             there are only five unique positions. This suggests to store five
+             unique vertices.
              
-             A non-trivial example is a set with several polylines, each of which with
-             eventually different number of vertices. The array stores the vertex
-             identifiers in the order how the polylines are visited:
+             A non-trivial example is a set with several polylines. Assume that each
+             has a different number of vertices. The array stores the identifier of
+             the vertices in the sequence how the polylines are visited:
              
-             The first entry is the identifier of the start vertex of the first polyline,
+             The first entry is the identifier of the first vertex of the first polyline,
              followed by the second vertex of the first polyline, until the last vertex
-             of the polyline. Thereafter, the start vertex of the second polyline, and
-             so on and so forth. Using the (cumulated) counts in number_of_vertices,
-             the vertices of the n-th polyline can be accessed on the following 
-             array index interval:
+             of the first polyline.
+             Thereafter, the first vertex of the second polyline, and so on and so forth.
+             Using the (cumulated) counts in number_of_vertices, the vertices of the
+             n-th polyline can be accessed on the following array index interval:
              :math:`[\sum_{i=0}^{i=N-1}, \sum_{i=0}^{i=N}]`.
```

Comment:
The summand `number_of_vertices` is missing

Reply:
Fixed